assets/js/forms/default-editor/form.js

```diff
@@ -29,7 +29,7 @@ export class Form {
 			} );
 
 			$el.replaceWith( matMenu.render() );
-			$( matMenu.select.menu_.root_ ).addClass( 'mdc-menu--supports-icon' );
+			$( matMenu.select.menu.root ).addClass( 'mdc-menu--supports-icon' );
```

Comment:
In assets/js/builder/notice/intro/notice.js the spaces between parenthesis were removed but here they are kept....just wanted to know which is correct or preferred for us.